<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>745</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>745</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Abbonamento</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>495</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GestisciAbbonamento.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>571</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="list_view"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_inserisci">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>INSERISCI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_elimina">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>ELIMINA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sospendi">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>SOSPENDI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_attiva">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>ATTIVA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="GestisciAbbonamenti.qrc"/>
 </resources>
 <connections/>
</ui>
